dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/entities/DataRequestEntity.kt

```diff
@@ -84,6 +95,14 @@ data class DataRequestEntity(
         }
     }
 
+    /**
+     * Adds a requestStatusEntity to the dataRequestStatusHistory.
+     * Note, this is not automatically saved in the database, you also need to persist the requestStatusEntity.
+     */
+    fun addToRequestStatusHistory(requestStatusEntity: RequestStatusEntity) {
```

Comment:
see above. I'd prefer "addToDataRequestStatusHistory" here. it's more consistent (e.g. to the doc-string) and doesn't add many characters

Reply:
Geändert